libcxx/include/limits

```diff
@@ -250,10 +239,8 @@ protected:
   static _LIBCPP_CONSTEXPR const int digits       = 1;
   static _LIBCPP_CONSTEXPR const int digits10     = 0;
   static _LIBCPP_CONSTEXPR const int max_digits10 = 0;
-  static _LIBCPP_CONSTEXPR const type __min       = false;
-  static _LIBCPP_CONSTEXPR const type __max       = true;
```

Comment:
I wonder why this ever existed. That dates from the initial import.

It could be that before we had push/pop macros for `min`/`max`, they used `__min` `__max` from within the library. I'm not sure, and it doesn't matter much anyways.